docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -28,9 +28,16 @@ public static void Example()
             Action<SamplesUtils.DatasetUtils.SampleInfertData, SampleInfertDataTransformed> mapping = 
                 (input, output) => output.IsUnderThirty = input.Age < 30;
 
-            // Custom transformations can be used to transform data directly, or as part of a pipeline. Below we transform data directly.
-            var transformer = mlContext.Transforms.CustomMappingTransformer(mapping, null);
-            var transformedData = transformer.Transform(trainData);
+            // CustomMappingEstimator can be used as part of a pipeline of estimators.
```

Comment:
>CustomMappingEstimator [](start = 15, length = 22)

This documentation will be on the `CustomMapping` transform catalog entry, so do we want to refer to the underlying `Estimator`? I might keep it as plain english or refer to the catalog entry.

Reply:
Sounds good. 

---
In reply to: [263481546](https://github.com/dotnet/machinelearning/pull/2876#discussion_r263481546) [](ancestors = 263481546)